### PR TITLE
Pc/ssl-error-message

### PR DESCRIPTION
**Pull request by Blaizzy**
Improved SSL error message

### CODE REVIEW

**Comment by Blaizzy** [comment]
I know that some checks will fail because of the new naming. It needs to be updated in all places where SSLError is called.

Let me know if it is something I can do.

File: neptune/api_exceptions.py [hunk 1]
```diff
@@ -21,12 +21,55 @@ class NeptuneApiException(NeptuneException):
...
+            {fail}"curl: (60) server certificate verification failed..."{end}
+    - Your machine software is not up-to-date.
+        - Minimal OS requirements:
+            - Windows >= XP SP3
+            - macOS >= 10.12.1
+            - Ubuntu >= 12.04
+            - Debian >= 8
+
+{bold}What can I do?{end}
+You can manually configure Neptune to skip all SLL checks. To do that
```
Comment (pkasprzyk) [inline neptune/api_exceptions.py:50]: `SLL` -> `SSL`

**Comment by pkasprzyk** [comment]
@Blaizzy a pro tip for the future: 
If you have test requirements installed (and you should ;) `pip install requirements/test_requirements.txt`) you can do: `pre-commit install`, which will install a git precommit hook for `black`. 
This way your code gets autoformatted each time you try to commit a change -- no more "trailing whitespace" or fomatting errors!

**Comment by Blaizzy** [comment]
@pkasprzyk thank you very much!

I realized that while working at it. I found out about pre-commit and installed it locally as well as turn on the trim-trailing-whitespace feature on vscode.

I was getting an error every single time. It seems like the pre-commit used in the GH action has problems removing trailing whitespaces automatically.

File: neptune/api_exceptions.py (hunk 1 above)
Comment (Blaizzy) [reply neptune/api_exceptions.py]: Thanks, it is fixed ✅

Comment (pkasprzyk) [reply neptune/api_exceptions.py]: it's not in the copy-pasted message of the second exception ;)

Comment (Blaizzy) [reply neptune/api_exceptions.py]: Can you elaborate?

Comment (Blaizzy) [reply neptune/api_exceptions.py]: No, it's not!

I did a complete search for SLL on all scripts just to double check.

Comment (Blaizzy) [reply neptune/api_exceptions.py]: 😅 It was, but I fixed it ✅ 

vscode global search couldn't match 'SLL' the second file 😞

**Review by pkasprzyk — APPROVED** [review]

**Pull request merged**